processing/threedidepth_algorithm.py

```diff
@@ -321,6 +330,21 @@ def processAlgorithm(self, parameters, context, feedback):
         else:
             timesteps = [parameters[self.CALCULATION_STEP_INPUT]]
 
+        raster_filename = parameters[self.WATER_DEPTH_LEVEL_NAME]
+        if not raster_filename:
+            raise QgsProcessingException(self.invalidSourceError(parameters, self.WATER_DEPTH_LEVEL_NAME))
+
+        output_location = parameters[self.OUTPUT_DIRECTORY]
+        if not output_location:
+            raise QgsProcessingException(self.invalidSourceError(parameters, self.OUTPUT_DIRECTORY))
+
+        as_netcdf = parameters[self.AS_NETCDF_INPUT]
+        raster_extension = "nc" if as_netcdf else "tif"
```

Comment:
`tif` or `tiff`? I don't know what's used most often?

Reply:
Raster format is `TIFF`, extension is `.tif`. :)